internal/manager/metadata/metadata.go

```diff
@@ -12,9 +12,17 @@ var (
 	// Release returns the release version, generally a semver like v2.0.0.
 	Release = "NOT_SET"
 
-	// Repo returns the git repository URL.
+	// Repo returns the git repository URL like git@github.com:Kong/kubernetes-ingress-controller.git.
 	Repo = "NOT_SET"
 
+	// ProjectURL returns address of project website - git repository like github.com/kong/kubernetes-ingress-controller.
+	ProjectURL = "NOT_SET"
+
 	// Commit returns the SHA from the current branch HEAD.
 	Commit = "NOT_SET"
 )
+
+const (
+	Organization = "Kong"
+	ProjectName  = "kubernetes-ingress-controller"
```

Comment:
Maybe instead of those hardcoded values we could introduce something like https://github.com/kubescape/go-git-url to try and parse the `Repo`?

Reply:
I believe we could do this yeah